cmd/activity-monitor/main.go

```diff
@@ -88,13 +90,43 @@ func startMonitor(rpcCh *amqp.Channel, logger *blog.AuditLogger) {
 		cmd.FailOnError(err, "Could not subscribe to queue")
 	}
 
+	deliveryTimings := make(map[string]time.Time)
+
 	// Run forever.
 	for d := range deliveries {
+		// If d is a call add to deliveryTimings and increment Boulder.RpcCallsWaiting, if it is a 
+		// response then get time.Since original call from deliveryTiming, send timing metric, and
+		// decrement Boulder.RpcCallsWaiting
+		go func() {
```

Comment:
I'd prefer it if you moved the delivery timings work into its own function, so that this one can remain primarily suited for dispatch.